### PR TITLE
Introduced SmartCash cronjobs to improve stability of the node

### PR DESCRIPTION
**Pull request by lapistano**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -5,14 +5,17 @@ ARG _smartcashBin=/opt/smartcash/smartcashd
 ARG _entryPointBin=/opt/docker-entrypoint.sh
 
 ENV WALLET_CONF /etc/smartcash/smartcash.conf
-ENV WALLET_DATA /data/
+ENV WALLET_DATA /data
+ENV WALLET_RUN /run/smartcash
 
 RUN apt-get update && \
+    apt-get upgrade -y  && \
```
Comment (tunikum) [inline Dockerfile]: `apt-get upgrade` is bad practice according to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run

File: Dockerfile [hunk 2]
```diff
@@ -21,10 +24,14 @@ RUN mkdir -p `dirname $WALLET_CONF` && \
     chmod +x $_entryPointBin && \
     ln -s $_entryPointBin /usr/local/bin/docker-entry
 
-VOLUME /data
+RUN mkdir -p `dirname $WALLET_RUN`
+
+ADD /cronjobs.tgz  $WALLET_RUN
```
Comment (tunikum) [inline Dockerfile]: Does it have to be a `tgz`? Shouldn't `COPY bin $WALLET_RUN` do it? Then it wouldn't be necessary to check in a binary file.

File: openfw.sh [hunk 3]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (tunikum) [inline openfw.sh]: This file seems to be unnecessary.

File: Dockerfile [hunk 4]
```diff
@@ -21,10 +24,14 @@ RUN mkdir -p `dirname $WALLET_CONF` && \
     chmod +x $_entryPointBin && \
     ln -s $_entryPointBin /usr/local/bin/docker-entry
 
-VOLUME /data
+RUN mkdir -p `dirname $WALLET_RUN`
```
Comment (tunikum) [inline Dockerfile]: This shouldn't be necessary.

**Review by tunikum — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (lapistano) [reply Dockerfile]: I c .. kk will change

**Pull request merged**